docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -1,22 +1,62 @@
 ---
 title: Joining Nodes in AWS
-description: How to join nodes and proxies on EC2
+description: How to join nodes and proxies running on AWS
 h1: Joining Node and Proxies in AWS
 ---
 
 This section will explain how to set up Teleport Nodes and Proxies to join your
-Teleport cluster with the EC2 join method. The EC2 join method enables you to
-allow Teleport nodes to join your Teleport cluster from EC2 instances in your
-AWS account without sharing any secrets.
+Teleport cluster without sharing any secrets when they are running in AWS.
+
+<Admonition type="note">
+You can still use secret tokens to join the cluster, see the
```

Comment:
Is this Admonition intended for users who are wondering whether the AWS join method replaces the token method? It might be more useful with more general information, e.g.,

```markdown
You can also configure Nodes to join a cluster via tokens, which is useful when <add a reason>. Read more in the following guide:

[Adding Nodes to the Cluster](../admin/adding-nodes.mdx)
```

Reply:
It is intended for readers who may see "Joining Nodes in AWS" and think this is the ONLY way to join nodes which are running in AWS, to make it clear that they can still use the usual token method. Will reword this part.